src/spline2/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# add apps XYZ.cpp, e.g., qmc_particles.cpp
```

Comment:
We need to tiny the many splines. This is a fine example of how to reduce accessibility of a code.